targets/unity-v2/templates/InstanceAPI.cs.ejs

```diff
@@ -0,0 +1,83 @@
+#if <%- getApiDefineFlag(api) %>
+using System;
+using System.Collections.Generic;
+using PlayFab.<%- api.name %>Models;
+using PlayFab.Internal;
+using PlayFab.Json;
+using PlayFab.Public;
+
+namespace PlayFab
+{
+<%- generateApiSummary("    ", api, "description")
+%>    public <%- isPartial %>class PlayFab<%- api.name %>InstanceAPI
+    {
+        public PlayFabApiSettings ApiSettings = null;
+        private PlayFabAuthenticationContext authenticationContext = null;
+
+        public PlayFab<%- api.name %>InstanceAPI() {}
+
+        public PlayFab<%- api.name %>InstanceAPI(PlayFabApiSettings settings) 
+        {
+            ApiSettings = settings;
+        }
+
+        public PlayFab<%- api.name %>InstanceAPI(PlayFabAuthenticationContext context) 
+        {
+            authenticationContext = context;
+        }
+
+        public PlayFab<%- api.name %>InstanceAPI(PlayFabApiSettings settings, PlayFabAuthenticationContext context) 
+        {
+            ApiSettings = settings;
+            authenticationContext = context;
+        }
+
+        public void SetAuthenticationContext(PlayFabAuthenticationContext context)
+        {
+            authenticationContext = context;
+        }
+
+        public PlayFabAuthenticationContext GetAuthenticationContext()
+        {
+            return authenticationContext;
+        }
+
+<% if (hasClientOptions) { %>
+        /// <summary>
+        /// Check to See if the client is logged in.
+        /// </summary>
+        public bool IsClientLoggedIn()
+        {
+            return authenticationContext == null ? false : authenticationContext.IsClientLoggedIn();
+        }
+<% }%>
+<% if (hasEntityTokenOptions) { %>
+        /// <summary>
+        /// Check to See if the entity is logged in.
+        /// </summary>
+        public bool IsEntityLoggedIn()
+        {
+            return authenticationContext == null ? false : authenticationContext.IsEntityLoggedIn();
+        }
+<% }%>
```

Comment:
In this case, you can remove these methods if they are not used anywhere.

Reply:
It's up to your decision on if (apiCall.auth === "SessionTicket").  As I asked you on previous comment

Reply:
Yeah if my replacements work, please remove these methods (if they are not used)

---
In reply to: [262707701](https://github.com/PlayFab/SDKGenerator/pull/420#discussion_r262707701) [](ancestors = 262707701)